locopy/redshift.py

```diff
@@ -600,23 +605,69 @@ def insert_dataframe_to_table(
             logger.info("New table has been created")
 
         logger.info("Inserting records...")
-        for start in range(0, len(dataframe), batch_size):
+        try:
+            length = len(dataframe)
+        except TypeError:
+            length = dataframe.select(pl.len()).collect().item()  # for polars lazyframe
```

Comment:
Same comment here on LazyFrame. If we want to consider not supporting that.

Same comment here on considering dispatch helper method to get `to_insert`.